spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/DockerCommandBuilderTests.java

```diff
@@ -17,14 +17,14 @@
 package org.springframework.cloud.deployer.spi.local;
 
 import static org.hamcrest.collection.IsArrayContainingInOrder.arrayContaining;
+import static org.hamcrest.Matchers.*;
 import static org.junit.Assert.*;
```

Comment:
Why aren't these expanded out?  Also, I know Framework's rules are to have static imports at the bottom of the import list: https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements

Reply:
we don't have checkstyle in place for this project....so just depends on local IDE settings.

I've merged this already, I can go in and optimize the imports and re-commit